src/dev/build/tasks/patch_native_modules_task.test.ts

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  ToolingLog,
+  ToolingLogCollectingWriter,
+  createAnyInstanceSerializer,
+  createAbsolutePathSerializer,
+} from '@osd/dev-utils';
+import { Build, Config } from '../lib';
+import { PatchNativeModules } from './patch_native_modules_task';
+
+const log = new ToolingLog();
+const testWriter = new ToolingLogCollectingWriter();
+log.setWriters([testWriter]);
+expect.addSnapshotSerializer(createAnyInstanceSerializer(Config));
+expect.addSnapshotSerializer(createAnyInstanceSerializer(ToolingLog));
+expect.addSnapshotSerializer(createAbsolutePathSerializer());
+
+jest.mock('../lib/download');
+jest.mock('../lib/fs', () => ({
+  ...jest.requireActual('../lib/fs'),
+  untar: jest.fn(),
+  gunzip: jest.fn(),
+}));
+
+const { untar } = jest.requireMock('../lib/fs');
+const { gunzip } = jest.requireMock('../lib/fs');
+const { download } = jest.requireMock('../lib/download');
+
+async function setup() {
+  const config = await Config.create({
+    isRelease: true,
+    targetAllPlatforms: false,
+    targetPlatforms: {
+      linux: false,
+      linuxArm: false,
+      darwin: false,
+    },
+  });
+
+  const build = new Build(config);
+
+  download.mockImplementation(() => {});
+
+  untar.mockImplementation(() => {});
+
+  gunzip.mockImplementation(() => {});
+
+  return { config, build };
```

Comment:
very minor: can we remove the extra whitespace here?

Reply:
I removed line 47 and 49